modules/programs/oh-my-posh.nix

```
@@ -47,6 +49,14 @@ in {
       '';
     };
 
+    configFile = mkOption {
+      type = types.nullOr types.str;
```

Can use a more specific type:
```suggestion
      type = types.nullOr types.path;
```

---

done